.github/workflows/ort.yml

```diff
@@ -215,14 +233,18 @@ jobs:
                     echo "No unapproved packages found."
                   fi
 
-              ### Check for attributions diff ###
+              ### Check for attributions diff ### (FFI attributions merged to FFI clients' attributions)
             - name: Check for diff
               run: |
                   cp python/ort_results/NOTICE_DEFAULT $PYTHON_ATTRIBUTIONS
                   cp node/ort_results/NOTICE_DEFAULT $NODE_ATTRIBUTIONS
                   cp glide-core/ort_results/NOTICE_DEFAULT $RUST_ATTRIBUTIONS
                   cp java/ort_results/NOTICE_DEFAULT $JAVA_ATTRIBUTIONS
-                  cp go/ort_results/NOTICE_DEFAULT $GO_ATTRIBUTIONS
+                  : Workaround for empty attributions for go packages. See discussions in https://github.com/valkey-io/valkey-glide/pull/3469 and in https://github.com/oss-review-toolkit/ort/issues/10454
+                  curl https://raw.githubusercontent.com/valkey-io/valkey-glide/9c9ef0ae2c2ee865c7867f6ac0d3f9aac054bac8/go/THIRD_PARTY_LICENSES_GO -o $GO_ATTRIBUTIONS
```

Comment:
I failed to ensure ORT to generate an attribution file for go for protobuf package dependency.
I asked for a help from ORT community in https://github.com/oss-review-toolkit/ort/issues/10454, but no luck so far. It is a release blocker, so as we discussed, I added this dirty trick to ensure that the attribution file contains license of protobuf go package.